mpc/.pylintrc

```diff
@@ -16,6 +16,11 @@ disable=
     protected-access,
     fixme,
     import-outside-toplevel,
+    unsubscriptable-object,
+
+# TODO: `unsubscriptable-object` generates false positives for python 3.9 and pylint==2.6.
+#   https://github.com/PyCQA/pylint/issues/3882
+# Re-enable it when the issue is fixed.
```

Comment:
The PR: https://github.com/PyCQA/pylint/pull/3890 was merged 2 days ago. Does it fully fix the issue you faced here?

Reply:
Yes, it appears that should fix it.  I'll watch the repo for the next release and try upgrading when it's available.

Reply:
Thanks